src/modes/overview/vegaSpec.js

```diff
@@ -159,7 +170,7 @@ export function createSpec(sensor, selections, dateRange, valuePatch) {
             field: yField,
             type: 'quantitative',
             scale: {
-              domainMin: valuePatch && valuePatch.domain ? scalePercent(valuePatch.domain[0]) : 0,
+              domainMin: valuePatch && valuePatch.domain ? scalePercent(valuePatch.domain[0]) : undefined,
```

Comment:
That was probably fine.  I left this change in accidentally, since I was trying to get rid of showing the tooltip (showing 0 as the value) for undefined values.